### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by NJ-2020**
Dear Reviewer,

We (Me and Moses) add a mobile menu navbar to this portfolio using HTML, CSS, and JavaScript.
We add a clickable Hamburger icon as requested.  When clicked, the menu appears.
We also make it responsive.

Thanks
Novell and Moses

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,26 +23,37 @@
...
+        <li>
+          <a href="#">Portfolio</a>
+        </li>
+        <li>
+          <a href="#">About</a>
+        </li>
+        <li>
+          <a href="#">Contact</a>
+        </li>
       </ul>
```
Comment (DeliceLydia) [inline index.html]: - [ ] Currently when I click one option of the mobile menu it won't disappear kindly fix that and I suggest that you link each list element with a related section not using dead links.

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @NJ-2020😊 , 

Good job so far! your hamburger button is currently working, but there are some issues that you still need to work on to go to the next project but you are almost there! Major things that I need to highlights:

- Set up linters correctly
- PR is descriptive
- Followed JavaScript best practices.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Using flexbox on your project, and contact sections on the desktop view so that they can fill all the empty space to make your user experience so memorable.

### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (m05e5) [reply index.html]: oh year thanks

**Comment by NJ-2020** [comment]
Dear Reviewer,

We fix the selector of the mobile menu option as suggested.
We also add JS linters check

Thanks

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html]: Well addressed

File: index.html [hunk 2]
```diff
@@ -248,23 +280,31 @@ <h2>About<br />Myself</h2>
...
           </li>
         </ul>
       </div>
     </section>
 
-    <section class="contact-section">
+    <section class="contact-section" id="contact">
       <form action="https://formspree.io/f/mayajwnb" method="post">
         <h2 class="contact-me-title">Contact Me</h2>
         <p class="contact-me-info">
```
Comment (Temmarie) [inline index.html:310]: - [ ] Kindly note that your project has to closely follow the template design in Figma. Refer to the template and make the required changes. 
- The hamburger icon on the navigation bar is slightly different from the one in the design. The logo and menu icon should be horizontally aligned.
- The project cards have some of their styles scrambled. Kindly fix them.
- The contact form uses the wrong background colour and is missing the required background image decoration.
- You have a horizontal overflow on mobile screens.
_See the images below for more context._

**Original** | **Your clone**
  ----------- | -----------
  image | image
 image | image
image | image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @NJ-2020 and @m05e5    ! :wave: :wave:

#### You have done really well in implementing the previously required changes.
 However, It seems while making those changes, you might have broken some things in the project. Kindly follow the suggestions below for this project to be approved. 

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. :smile:
- - -

### **Required Changes** ♻️

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 1 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi Novel and Moses! :wave: :wave:

####  Your project is complete! Nice job done for implementing the changes requested by the previous reviewer(s) :clap:  :clap: Your mobile menu is working correctly and your portfolio looks great! Congratulations 🎉🍾🍾 . There are however a few recommendations to implement to improve your project. Kindly check the optional comment below : 👍

## Good points 👍
No Linters errors ✔️
Semantics and syntax  are well used✔️

#### You may merge, good luck on your next project! :rocket:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**

- [OPTIONAL] Your mobile menu items appear to be widely spaced vertically. Also, when in mobile menu view the right margin ap the right margin has a wide gap which makes the margins on both the left and right uneven.
Your design looks like this in mobile view:
shot5

The original design looks like this;

shot4

Let's improve our design by adding equal margins at both left and right sides, and between the menu items


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html:310]: Not addressed

**Comment by nidalaa** [comment]
Hello @NJ-2020  and  @m05e5 ! 👋 

We noticed that you both requested a review for the same repo. 

Please note that for this project you should implement changes in 2 repos. Thanks to that both of your portfolios will be fully functional.

Check the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md):


>You need to use pair-programming for this project. We will check to see if both partners submitted code by looking at the commit history for each project. Therefore you need to implement the above features in all of the portfolios in your pair-programming group.


That will be also valid for the next 2 projects. Please pay attention to their descriptions.

For this project
- @NJ-2020  your project is already approved.
- @m05e5  please ask for review again but with a link to the PR in your portfolio's repo https://github.com/m05e5/moses_portfolio :)

**Pull request merged**